src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -50,22 +63,55 @@ public String getUrlForPath(String wikiPath) {
   }
 
   public String getUrlForWiki() {
-    return getUrlForWiki(Configuration.getWikiName());
+    return getUrlForWiki(Configuration.getWikiName(), false);
   }
 
   public String getUrlForWiki(String wikiName) {
-    String prefix = "wikia".equals(wikiName) ? "www." : "";
-    String suffix;
+    return getUrlForWiki(wikiName, false);
+  }
 
-    if (env.contains("dev") && !env.contains(SANDBOX_MERCURY_DEV)) {
-      String devBoxOwner = env.split("-")[1];
+  public String getUrlForWiki(boolean addWWW) {
+    return getUrlForWiki(Configuration.getWikiName(), addWWW);
+  }
 
-      suffix = "." + devBoxOwner + "." + "wikia-dev.com";
-    } else {
-      suffix = wikiName.endsWith("wikia") ? ".com" : ".wikia.com";
+  public String getUrlForWiki(String wikiName, boolean addWWW) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getUrlForWiki" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)